src/primitives/rectangle.js

```diff
@@ -2,19 +2,20 @@ const vec2 = require('../math/vec2')
 
 const geom2 = require('../geometry/geom2')
 
-/** Construct an axis-aligned rectangle with four sides and four 90-degree angles.
+/**
+ * Construct an axis-aligned rectangle with four sides and four 90-degree angles.
  * @param {Object} [options] - options for construction
  * @param {Array} [options.center=[0,0]] - center of rectangle
- * @param {Array} [options.size=[1,1]] - size of rectangle, width and height
+ * @param {Array} [options.size=[2,2]] - full dimension of rectangle, width and length
```

Comment:
I do not think it is necessary to specify 'full dimensions' here, that is the logical assumption when you say width and length I think :)

Reply:
Done